### DISCUSSION TITLE
[SOLVED] - Creating a Production Server but can not send email because Google disable [Less secure app access]

### DISCUSSION THREAD

**Discussion by KSB-tqk** [saran-surya/email_auth#68] [Q&A]
For those who had created a production server but can not send email because Google disable the [Less secure app access]
Check out this video:  
https://www.youtube.com/watch?v=uVDq4VOBMNM&t=29s&ab_channel=SysTools
- Hope it help

**Comment by saran-surya** [comment]
This does help, Will check the usage in local and update the documentation accordingly, 
Thankyou @KSB-tqk

**Comment by saran-surya** [comment]
### Kindly use test server as of now.
Remote server patches are on the way, expect in 7 days

This should give you a good lead.
https://github.com/saran-surya/email_auth#update--google-turning-off-less-secure-apps--switching-to-oauth
